### PR TITLE
Adding rsconnect-python

### PR DESCRIPTION
**Pull request by bcwu**
This PR adds rsconnect-python as a conda-forge feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/rsconnect-python/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
```
Comment (dopplershift) [inline recipes/rsconnect-python/meta.yaml:22]: Upstream `install_requires` says `click >=7.0.0` and `six >=1.14.0`. Any reason not to match?

Comment (bcwu) [reply recipes/rsconnect-python/meta.yaml:22]: Is the correct way to mirror over the requirements.txt over to host? and also to run if we expect the user as host?

Comment (bcwu) [reply recipes/rsconnect-python/meta.yaml]: added with https://github.com/conda-forge/staged-recipes/pull/14506/commits/613a5adbc78caeeb112827af85c80150dcf92815

**Comment by bcwu** [comment]
Per guidelines, commenting to confirm that I am willing to be listed as maintainer.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**